outlines/methods.md

```diff
@@ -0,0 +1,43 @@
+# Methods
+
+1. What are Methods?
+  1. Methods are your app's data input API methods, like a POST request in REST
+  2. One of the most advanced RPC systems for the web, with lots of "hidden" benefits that make your life easier
+  2. Implementation in DDP
+    1. Method simulation executed on client
+    2. Method message sent over DDP: name, arguments
+    3. Method executed on the server
+    4. Return value sent to the client
+    5. After all DB operations are complete and published, updated message sent to the client
+    6. Method callback fires with the result
+  3. Methods can be used for data loading, but have disadvantages compared to publications
+  4. Benefits of the Method system
+    1. Non-blocking but synchronous-looking (other methods from the same client won't run at the same time unless you use this.unblock)
+    2. Methods always execute in the same order, and results come back in the same order
+    3. Deeply optimized for optimistic UI because of change tracking
+2. Defining a method
+  1. Simple Meteor core version
+  2. Meteor core version with boilerplate
+  3. mdg:validated-method version
+  4. What it means to define a method on the server only vs. client and server
```

Comment:
Add a historical note about allow/deny, which is basically "completely ignore it".